addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/api/IBaseFroniusResponse.java

```diff
@@ -0,0 +1,11 @@
+package org.openhab.binding.fronius.internal.api;
+
+/**
+ * Basic interface for a response-object from the API
+ * 
+ * @author rokohl
```

Comment:
Should be `@author Thomas Rokohl - Initial contribution` and remove empty line on next row.